go/worker/compute/executor/committee/node.go

```diff
@@ -753,14 +784,19 @@ func (n *Node) handleScheduleBatch(force bool) {
 
 	// Ask the scheduler to get us a scheduled batch.
 	batch := n.scheduler.GetBatch(force)
-	if len(batch) == 0 {
+	n.logger.Debug("got a batch",
+		"batch", batch,
```

Comment:
This seems verbose.

Reply:
Yeah this one is not as useful and very verbose. Removing it.